src/components/VideoPlayer/videoActions/BottomBar/SecondaryVideoActions/SecondaryVideoActions.tsx

```diff
@@ -0,0 +1,40 @@
+import { BtnFullScreen, BtnMute, BtnNextTrack, BtnPlay, BtnPrevTrack, BtnRepeat } from "components/VideoPlayer/videoActions/buttons";
```

Comment:
dont try to break things to this level.
Like you could have written this code in single file under the videoActions folder.
You can write multiple components in a single file